datadog_checks_downloader/README.md

```diff
@@ -50,6 +56,28 @@ To run the tests, [install tox][8] and just run:
 tox
 ```
 
+You can select between online and offline tests when running testsuite using
+pytest:
+
+```shell
+pytest -vvvv -m online     # Run tests that use data from publicly a accessible repository.
```

Comment:
Integrations are typically tested using `ddev test ____`. Is there a way to run your tests with that instead of directly calling `pytest`?

Reply:
Yes, `ddev test datadog_checks_downloader` should work as expected. I can include it in the README file if necessary - I just continued with what is already written in the README.